packages/astro/src/runtime/server/jsx.ts

```diff
@@ -288,7 +288,7 @@ function filteredConsoleError(msg: any, ...rest: any[]) {
 		// the following error, which we can safely filter out:
 		const isKnownReactHookError =
 			msg.includes('Warning: Invalid hook call.') &&
-			msg.includes('https://reactjs.org/link/invalid-hook-call');
+			msg.includes('https://react.dev/warnings/invalid-hook-call-warning');
```

Comment:
I don't think React has updated their error message links yet, so we should probably revert this: https://github.com/search?q=repo%3Afacebook%2Freact%20invalid-hook-call&type=code